### DISCUSSION TITLE
Are external libraries a must have feature?

### DISCUSSION THREAD

**Discussion by poisonnuke** [immich-app/immich#11706] [Q&A] [answered]
Due to the HIGHLY concerning info from bo0tz about a potential removal of the external library functionality I would like to open a discussion if that is really in the interest of all users? 

There should be at least a survey under all users. Ive paid for immich and without external library immich would be COMPLETELY useless for me. 

Im surely not the only one to use immich in conjunction with other tools?

**Comment by bo0tzz** [comment, marked as answer]
I'm not sure where you're coming from, nowhere did I say that external libraries are being removed.

**Reply by poisonnuke** [reply]
like 30 min ago:
https://github.com/immich-app/immich/discussions/11705#discussioncomment-10307206

**Reply by bo0tzz** [reply]
That doesn't say we're removing external libraries, it says that we're trying to remove their special treatment.

**Reply by poisonnuke** [reply]
youre way too quick on closing discussions though. And the answer in the other topic was far from clear.  You should stop closing topics like that!

**Reply by mmomjian** [reply]
I closed this discussion - you misunderstood a simple answer and instead of replying you made a new thread based on fear, which is not necessary. Please keep discussion in-line with the thread in the future. Thank you.

**Reply by poisonnuke** [reply]
I opened new discussion because the old one was closed before I could reply. I would have loved to clearify in the old topic but I was not allowed to.

**Reply by mmomjian** [reply]
You can still reply to closed topics, like this one :)

**Reply by poisonnuke** [reply]
Im also wondering, ive tried the same thing over in the other and the reply button did not turn green. Your still pretty quick on closing though ;)

**Reply by alextran1502** [reply]
We are trying to resolve anything as soon as we can, otherwise, things would get unanswered and lost in the increasing amount of inquiries and questions.
